src/env.js

```diff
@@ -59,6 +62,12 @@ export const apis = Object.freeze({
     /** Whether the WebNN API is available */
     IS_WEBNN_AVAILABLE,
 
+    /** Symbol from JS environment that exposes their own ONNX runtime */
+    EXPOSED_RUNTIME_SYMBOL,
+
```

Comment:
```suggestion
```

Maybe one question - is there a reason this should be exposed via `apis`? It isn't used elsewhere

Reply:
nvm - it is!